index.html

```diff
@@ -59,25 +59,55 @@
     </div>
   </nav>
   <section id="about-us">
-    <div class="card">
-      <div class="row g-0">
-        <div class="col-md-4">
-          <img src="\img\waw.png" class="img-fluid rounded-start" alt="Group's photo">
+    <div class="container">
```

Comment:
Since the padding and margin is custom, remove this `div` (but not the contents of it, move them to be childs of `section` ).